src/lib.rs

```diff
@@ -0,0 +1 @@
+mod sg;
```

Comment:
when we create this mod, it will try to load sg/mod.rs? and then inside of mod.rs we define the interface of the app (Let's say: the CLI module, that we talked before) ?